src/flow/store/event/eventActionLazy.js

```diff
@@ -403,3 +404,55 @@ export function removeConferencingRoom(url) {
     setTimeout(() => dispatch(removeConferencingRoomAction('removed')), 1000)
   }
 }
+
+export function verifyCalendarAction(payload) {
+  return { type: VERIFY_ADD_CALENDAR, payload }
+}
+
+export function verifyAddCalendar(calendar) {
+  console.log('verifyCalendar')
+  return async (dispatch, getState) => {
+    if (calendar == null) {
+      verifyCalendarAction({ status: '' })
+      return
+    }
+
+    dispatch(
+      verifyCalendarAction({
+        calendar,
+        status: 'pending',
+      })
+    )
+
+    await importCalendarEvents(calendar, defaultEvents).then(
+      events => {
+        const calendarEvents = {
+          name: calendar.name,
+          events,
+        }
+        console.log('import ok', calendarEvents)
+        dispatch(
+          verifyCalendarAction({
+            status: 'ok',
+            calendarEvents,
+            eventsCount: Object.keys(events).length,
+          })
+        )
+      },
+      error => {
+        const msg = 'failed to verify calendar'
+        console.log(msg, error)
+        dispatch(verifyCalendarAction({ status: 'error' }))
```

Comment:
The error should be transmitted in the result as well.

Reply:
Done.